CHANGELOG.md

```
@@ -1,3 +1,13 @@
+# [1.24.0](https://github.com/misty-step/landfall/compare/v1.23.1...v1.24.0) (2026-06-15)
+
+### Features
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)